arlo-client/src/components/Audit/useJurisdictions.ts

```diff
@@ -40,6 +40,18 @@ export interface IJurisdiction {
   } | null
 }
 
+export const prettifyStatus = (processing: IFileInfo['processing']): string => {
+  if (!processing) return 'No manifest uploaded'
+  switch (processing.status) {
+    case FileProcessingStatus.ERRORED:
+      return `Manifest upload failed: ${processing.error}`
```

Comment:
Have you tried this case out? I'm curious what the table looks like if the error is long. Does it render well?

Reply:
@jonahkagan Yup, I threw in a massive long string, and the dialog shows it beautifully. In the table it simply truncates it with an ellipses at the end, so it works nicely there as well (since the same error will be in the details in the dialog box, so there's no need to dump the whole thing visibly in the table).

Reply:
Awesome